.gitignore

```diff
@@ -41,3 +41,4 @@ nb-configuration.xml
 .sdkmanrc
 .envrc
 .jekyll-cache
+vale.ini
```

Comment:
vale.ini should not be ignored unless/until we move things to .vale.ini, which @maxandersen may change in #29913

Reply:
@ebullient - Oops sorry  :blush: Didn't mean to commit that. Commit undone. The joys of working on different docs with different rulesets!